contrib/forms/psychiatrySet/intakeverslag/view.php

```diff
@@ -204,7 +204,7 @@ function autosave( )
 
 
 <form method=post action="<?php echo $rootdir?>/forms/intakeverslag/save.php?mode=update&id=<?php echo attr_url($_GET["id"]); ?>" name="my_form">
-<span class="title"><?php echo xlt('Psychiatric Intake'); ?></span><Br><br>
+<span class="title"><?php echo xlt('Psychiatric Intake'); ?></span><Br><br />
```

Comment:
bad Br tag here also (note I am only skimming this commit, but somehow noted this :) )